photocollage/gtkgui.py

```diff
@@ -20,6 +20,10 @@
 import cairo
 import copy
 import gettext
+import gi
+
+gi.require_version('Gtk', '3.0')
```

Comment:
Could you place this after the imports (in the same way it is done for `gettext`)?